pkg/reconciler/revision/resources/queue.go

```
@@ -191,13 +226,23 @@ func makeQueueContainer(rev *v1alpha1.Revision, loggingConfig *logging.Config, o
 		volumeMounts = append(volumeMounts, internalVolumeMount)
 	}
 
+	rp := rev.Spec.GetContainer().ReadinessProbe.DeepCopy()
+
+	applyReadinessProbeDefaults(rp, userPort)
+
+	probeJSON, err := json.Marshal(rp)
+	if err != nil {
+		probeJSON = []byte{}
```

🤔  I don't like swallowing errors :(